wkcuber/export_wkw_as_tiff.py

```diff
@@ -0,0 +1,171 @@
+from argparse import ArgumentParser
+
+import logging
+import wkw
+import os
+import numpy as np
+from PIL import Image
+from typing import Tuple, Dict
+
+from .metadata import read_metadata_for_layer, read_datasource_properties
+from .utils import add_verbose_flag, add_distribution_flags, get_executor_for_args
+from .mag import Mag
+
+
+def create_parser():
+    parser = ArgumentParser()
+
+    parser.add_argument(
+        "--source_path", "-s", help="Directory containing the wkw file.", required=True
+    )
+
+    parser.add_argument(
+        "--destination_path",
+        "-d",
+        help="Output directory for the generated dataset.",
+        required=True,
+    )
+
+    parser.add_argument(
+        "--layer_name",
+        "-l",
+        help="Name of the layer that will be converted to a tiff stack",
+        default="color",
+    )
+
+    parser.add_argument("--name", "-n", help="Name of the tiffs", default=None)
+
+    parser.add_argument(
+        "--axis",
+        "-a",
+        help="The axis that the image should be generated along. "
+        "Thus choosing z will print x,y slices.",
```

Comment:
On a second thought, I'm not sure whether we really need this feature. Did you have a specific use case in mind when you wrote the code? I think, using z by default is sufficient for us (the cubing process also always assumes z as the default axis). I cannot think of a reason to use another axis :thinking:

Reply:
I thought this tool is used to export some part of a wkw-file to tiff files and since you might not always just look at x,y-slices, I decided to add that feature.